.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
It doesn't seem to be running, maybe because the branch is the `main` branch?

Reply:
The incorrect branch name was fixed, but the CI still doesn't run. Maybe it requires to be merged for the first time.

Besides, the CI workflow should be triggered for pull requests on any branch.